docs/03-apis/api-v2/reading-and-searching-resources.md

```diff
@@ -571,22 +571,26 @@ be sorted alphabetically by resource IRI (an arbitrary but consistent order).
 The value of `page` is a 0-based integer page number. Paging works as it does
 in [Gravsearch](query-language.md)).
 
-### Get the Version History of Resources and Values of a Project
+### Get the Full History of a Resource and its Values as Events
 
-To get a list of the changes that have been made to resources and values of a project since their creation ordered by date
-use this route:
+To get a list of the changes that have been made to a resource and its values since its creation as events ordered by 
+date:
 
 ```
-HTTP GET to http://host/v2/resources/projectHistory/projectIRI
+HTTP GET to http://host/v2/resources/resourceHistoryEvents/<resourceIRI>
 ```
 
-The project IRI must be URL-encoded. The response is a list of events describing changes made to the resource and its values,
+The resource IRI must be URL-encoded. The response is a list of events describing changes made to the resource and its values,
  in chronological order. Each entry has the properties: 
  `knora-api:eventType` (the type of the operation performed on a specific date. The operation can be either
- `createResource`, `deleteResource`, `createValue`, `updateValueContent`, `updateValuePermissions`, or `deleteValue`.), 
+ `createResource`, `updateResourceMetadata`, `deleteResource`, `createValue`, `updateValueContent`, `updateValuePermissions`, 
```

Comment:
As a general remark when naming events, the convention (in the event based world) is for them to have a name in the past tense, e.g., ResourceCreated, ResourceDeleted, etc. They denote something that has happened.